### DISCUSSION TITLE
Fetch inside a function inside load

### DISCUSSION THREAD

**Discussion by Gustutu** [sveltejs/kit#11197] [Q&A]
I want to call the load fetch function somewhere in my code. Do i have to pass it as argument down to the place i want to use it or is there a way to call it from anywhere in the code without having to pass it as an argument.
Am i right to say that if i am not passing it as argument and i call fetch it will not be the right "load fetch" function but the "classic fetch" function ?

**Comment by Angus-Paillaugue** [comment]
Yes, if you want to use the load fetch function you can pass it as an argument like so : 

```
export const load = async ({ fetch }) => {

	return {
		fetcher: fetch
	};
};
```
And if you want to use it throughout your page, you can redefine it by doing `window.fetch = fetcher`
